src/cr/cube/stripe/cubemeasure.py

```diff
@@ -85,6 +88,10 @@ class _CatCubeMeans(_BaseCubeMeans):
     @lazyproperty
     def means(self):
         """1D np.float64 ndarray of mean for each stripe row."""
+        if self._means is None:
+            raise ValueError(
+                "`.means` is undefined for a cube-result without a means measure"
+            )
```

Comment:
As I think more about this, I'm thinking that the exception-raising should move upwards, like into the factory, rather than have it down here where it needs to be repeated for each subclass. There's no way the factory can produce a valid `CubeMeans` object if `cube.means` is `None`, right? So might as well move it up there and reduce the number of times these four lines appear repeated in the code.

I'm _almost_ thinking that we should move it further, right up to `Cube.means`, but that would complicate testing for whether the cube-result had a means measure; probably better to just move it up the one level to the factory, and then any `_BaseCubeMeans` subclass can rely on its `._means` instance variable to contain a valid cube-means array.